src/sage/matroids/matroid.pyx

```diff
@@ -8159,23 +8337,21 @@ cdef class Matroid(SageObject):
         matroids.insert(0, self)
         return union_matroid.MatroidUnion(iter(matroids))
 
-    def direct_sum(self, matroids):
+    cpdef direct_sum(self, matroids) noexcept:
         r"""
         Return the matroid direct sum with another matroid or list of
         matroids.
 
         Let `(M_1, M_2, \ldots, M_k)` be a list of matroids where each `M_i`
-        has ground set `E_i`. The matroid sum `(E_1,I_1),\ldots,(E_n,I_n)`
+        has groundset `E_i`. The matroid sum `(E_1,I_1),\ldots,(E_n,I_n)`
```

Comment:
should this not say the "matroid sum of ... is"?

Or use an operator instead of commas?